config/test-oak-camera.json

```diff
@@ -5,20 +5,24 @@
       "app": {
           "driver": "osgar.drivers.oak_camera:OakCamera",
           "init": {
-            "fps": 5,
+            "fps": 10,
             "is_color": true,
             "is_depth": true,
-            "cam_ip": "192.168.1.51",
+            "laser_projector_current": 1200,
+            "is_imu_enabled": true,
+            "number_imu_records": 10,
+            "disable_magnetometer_fusion": false,
```

Comment:
I do not like the inverse logic (you have to think about it too much - why not `use_magnetometer_fusion`? (with default set in code, maybe to `True`?)

Reply:
The logic is that the default is all sensors together and a option is disable some sensor (e.g. magnetometer).